analytics/models.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import, division, unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Analytics(models.Model):
+    page = models.OneToOneField('wagtailcore.Page')
+    last_week_views = models.PositiveIntegerField(default=0)
```

Comment:
We talked about making the period configurable as a site default, in case a week wasn't the best option.  Rename to last_period_views or something more generic like that.